Cakefile

```diff
@@ -161,41 +177,70 @@ task 'doc:site', 'watch and continually rebuild the documentation for the websit
       button = if executable then """<div class="minibutton ok" onclick="javascript: #{js.replace /"/g, '&quot;'};#{append}">#{run}</div>""" else ''
       "<div class='code'>#{cshtml}#{jshtml}#{script}#{load}#{button}<br class='clear' /></div>"
 
-  monthNames = [
-    'January'
-    'February'
-    'March'
-    'April'
-    'May'
-    'June'
-    'July'
-    'August'
-    'September'
-    'October'
-    'November'
-    'December'
-  ]
-
-  formatDate = (date) ->
-    date.replace /^(\d\d\d\d)-(\d\d)-(\d\d)$/, (match, $1, $2, $3) ->
-      "#{monthNames[$2 - 1]} #{+$3}, #{$1}"
-
-  releaseHeader = (date, version, prevVersion) -> """
-    <div class="anchor" id="#{version}"></div>
-    <b class="header">
-      #{prevVersion and "<a href=\"https://github.com/jashkenas/coffeescript/compare/#{prevVersion}...#{version}\">#{version}</a>" or version}
-      <span class="timestamp"> &mdash; <time datetime="#{date}">#{formatDate date}</time></span>
-    </b>
-  """
+  htmlFor = ->
+    marked = require 'marked'
```

Comment:
Done. Are you okay with me merging this into develop whenever I'm done testing it? I'm working on getting the 2 docs built in their branch, based on this.